app/models/user.rb

```diff
@@ -1968,7 +1968,7 @@ def set_default_categories_preferences
       end
     end
 
-    CategoryUser.insert_all!(values) if values.present?
+    CategoryUser.insert_all(values) if values.present?
```

Comment:
I find out that if you didn't specify the `unique_key:` param then it will use the default unique key available in the table.